extension/src/repository/index.ts

```diff
@@ -45,7 +48,10 @@ export class Repository extends DeferredDisposable {
   }
 
   public getState() {
-    return this.model.getState()
+    return {
+      ...this.model.getDecorationState(),
+      ...this.model.getSourceControlManagementState()
```

Comment:
[F] This function is only for testing. When I rewrite this I will get rid of the need for it.